### DISCUSSION TITLE
404 File not Found with create-react-app application

### DISCUSSION THREAD

**Discussion by AZensky** [community/community#33130] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I tried to deploy a simple application to github pages, but it shows 404 file not found. I've tried changing the folder to /docs and renaming the public folder to /docs.

**Comment by radar-blip** [comment, marked as answer]
You must have just fixed this?

**Reply by AZensky** [reply]
Not really, I couldn't figured it out, so I deployed it using Netlify. Thank you!

**Comment by jfullstackdev** [comment]
if you can share the exact link of that repo if that is public

it's hard to tell on your profile which repo is that

**Reply by AZensky** [reply]
Unfortunately I have now deleted the necessary code to deploy to gh pages, and deployed it using Netlify. Thank you though!
